wiki/Ranking_Criteria/osu!mania/id.md

```diff
@@ -0,0 +1,201 @@
+- **Pola braket harus digunakan secara hemat.**
+- **Long note yang dimainkan dengan key berdekatan yang dipegang secara bersamaan harus dilepaskan pada waktu yang bersamaan.** Long note dengan beberapa waktu rilis membutuhkan koordinasi tangan yang tidak dapat ditangani oleh pemain di level ini.
+
+### ![](/wiki/shared/diff/hard-m.png) Hard
+
+#### Pedoman
+
+- **HP dan OD tidak boleh lebih dari 8.**
+- **Note snapping 1/8 berturut-turut atau lebih tidak diperbolehkan.** Ini bisa mengakibatkan kesulitan untuk pemain pada level ini. Pengecualian untuk panduan ini adalah penggunaan 3 note yang berada pada snap high-end (1/8, 1/12, dan 1/16) karena hanya digunakan untuk menonjolkan suara gracing
+- **Stream panjang 1/4 harus disertakan, setidaknya memiliki jeda 1/2 setelah frase musik.** Ini bisa menyebabkan kesulitan untuk pemain pada level ini.
+- **Penempatan chord 1/4 setidaknya memiliki jeda 1/1 ketukan antara satu sama lain.** Penggunaan chord yang terlalu rapat akan menyebabkan kesulitan untuk pemain pada level ini.
+- **Hindari penggunaan chord di tengah 1/6 stream.** Namun, Anda masih bisa menggunakannya di awal atau akhir stream.
+- **Tidak disarankan menggunakan 1/4 jack.** Ini bisa menyebabkan kesulitan untuk pemain pada level ini.. Minijack dengan jeda yang cukup masih dapat diterima.
+- **Hindari penggunaan chord lebih dari 9 secara berurutan saat trill.** Koordinasi dan keseimbangan tangan yang diperlukan untuk pola ini akan menyebabkan kesulitan untuk pemain pada level ini.
+- **Penggunaan long note harus ditahan setidaknya 1/4 ketukan.**
+- **Objek langsung setelah perubahan slider velocity jangka pendek akan muncul pada scroll speed normal beatmap.** Untuk melakukan ini, perubahan slider velocity jangka pendek hanya boleh digunakan saat ruang kosong diantara nada dan harus disamakan dengan jarak 1,0x.
+- **Penggunaan slider velocity jangka panjang harus diantara 0,80x hingga 1,10x.**
+- **Gimik perubahan slider velocity harus bereaksi dalam 1/2 ketukan atau lebih.**
+
+Tambahan pedoman untuk tingkat kesulitan *4 key Hard*:
+
+
+- **Hindari chord lebih dari tiga note.** Ini juga termasuk long note yang melibatkan lebih dari tiga kolom.
+- **Hindari penggunaan anchor lima chord atau lebih.** Meskipun mudah dibaca oleh pemain, anchor yang lebih panjang akan menjadi lebih berat.
+
+Tambahan pedoman untuk tingkat kesulitan *7 key Hard*:
+
+
+- **Hindari penggunaan chord lebih dari empat.** Ini juga termasuk long note yang melibatkan empat kolom.
+- **Hindari penggunaan anchor lebih dari lima note atau lebih.** Meskipun mudah dibaca oleh pemain, anchor yang lebih panjang akan menjadi lebih berat.
+- **Jaga jarak chord pada chordstream pada interval 1/1 atau lebih.** Lebih rapat dari itu akan membuat sukar untuk dibaca.
+- **Tangan sesekali dapat digunakan dalam stream, asalkan dipisahkan setidaknya satu takaran (4 ketukan)**
```

Comment:
Ini harus dipikirin lagi kata2 yang tepat. Aku sendiri belum nemu juga tbh, buat kata2 yang pas nya.

Reply:
Fixed. Totally re-translate for 1st line and some edit for 4th line, rest is fine i guess. Also i've recheck it myself since there's some missing "." on Guidelines. Re-translate some awkward parts too.